src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px;
+
+  &__image {
+    align-self: center;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    display: inherit;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: inherit;
+
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+    line-height: 18px;
+  }
+
+  &__value {
+    color: $main-text-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: block;
+    margin-bottom: 16px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding